django_rq/queues.py

```diff
@@ -49,6 +49,7 @@ class DjangoRQ(Queue):
     def __init__(self, *args, **kwargs):
         autocommit = kwargs.pop('autocommit', None)
         self._autocommit = get_commit_mode() if autocommit is None else autocommit
+        self.exception_handlers = kwargs.pop('exception_handlers', None)
```

Comment:
Sorry, the test should be for `get_worker()` because only workers have `exception_handlers`. `Queue` objects have no `exception_handlers` in RQ.